src/SeleniumLibrary/keywords/window.py

```diff
@@ -20,7 +20,7 @@
 
 from SeleniumLibrary.base import keyword, LibraryComponent
 from SeleniumLibrary.locators import WindowManager
-from SeleniumLibrary.utils import plural_or_not
+from SeleniumLibrary.utils import (plural_or_not, is_string)
```

Comment:
Nit pic: Parenthesis hare are not needed.

Reply:
I'll will this